waku/protocol/v1/waku_mail.nim

```diff
@@ -9,22 +9,13 @@
 #
 import
   chronos,
-  eth/[p2p, async_utils],
-  ./waku_protocol
+  eth/[p2p, async_utils], eth/p2p/rlpx_protocols/whisper/whisper_types,
+  ./mailserver,
+  db_sqlite, ./waku_protocol
 
 const
   requestCompleteTimeout = chronos.seconds(5)
 
-type
-  Cursor = seq[byte]
-
-  MailRequest* = object
```

Comment:
If we use a `waku_mail` and `waku_mail_server` file, then I'd rather keep these in the `waku_mail` ones. Makes more sense I think to have a simple waku client to not have to import waku_mail_server.